locales/en/settings.ftl

```diff
@@ -8,9 +8,14 @@ settings-page-title = { -product-short-name } Settings
 
 ## Breach alert preferences
 
-settings-alert-preferences-title = Breach alert preferences
+settings-alert-email-preferences-title = Email preferences
+settings-alert-email-preferences-subtitle = Tell us which emails you’d like to receive.
+settings-alert-preferences-allow-breach-alerts-title = Instant breach alerts
+settings-alert-preferences-allow-breach-alerts-subtitle = These alerts are sent immediately once a data breach is detected
 settings-alert-preferences-option-one = Send breach alerts to the affected email address
 settings-alert-preferences-option-two = Send all breach alerts to the primary email address
+settings-alert-preferences-allow-monthly-monitor-report-title = Monthly { -brand-monitor } report
```

Comment:
Too late for a review 🤔 

Are these emails available for locales other than en-US?

Reply:
Not yet, but they will be. The non-monthly settings strings will be visible to everyone.